src/server.ts

```diff
@@ -129,12 +129,13 @@ DAO.connect(() => {
 
   //Routes
   const donorsRoute = require("./routes/donors");
-  const donationsRoute = require("./routes/donations");
+  const donationsRoute = require("./routes/donations").default;
```

Comment:
What's going on here?

Reply:
It seemed necessary while writing tests to make ES import/exports for type inference, which meant changing `module.exports = router` in each to `export default router`. However when requiring those modules, it was either necessary to replace it with an import statement on the top, or to add this `.default` thing